packages/scenario/test/repeat.test.ts

```diff
@@ -672,6 +672,592 @@ describe('Tests ported from JavaRosa - repeats', () => {
 				});
 			});
 		});
+
+		interface SubstituteAbsoluteBodyReferencesOptions {
+			readonly substituteAbsoluteBodyReferences: boolean;
+		}
+
+		describe('//region Deleting repeats', () => {
+			/**
+			 * **PORTING NOTES**
+			 *
+			 * - Rephrase "repeat group" -> "repeat instance"?
+			 */
+			describe('[deleting] delete second repeat group', () => {
+				/**
+				 * **PORTING NOTES**
+				 *
+				 * - Rephrase "triggerables"?
```

Comment:
I've rephrased a few of these in a commit, with the aforementioned hindsight approach of attaching the original JR name in a comment above. I didn't do a full pass, as it was getting tedious, but I noted in the commit note that it can serve as a reminder to revisit those as they come up.